web/packages/teleport/src/components/ExternalAuditStorageCta/ExternalAuditStorageCta.tsx

```diff
@@ -100,6 +87,41 @@ export const ExternalAuditStorageCta = () => {
   );
 };
 
+function CtaButton(props: { featureEnabled: boolean; userHasAccess: boolean }) {
+  if (!props.featureEnabled) {
+    return (
+      <ButtonLockedFeature
+        height="32px"
+        size="medium"
+        event={CtaEvent.CTA_EXTERNAL_AUDIT_STORAGE}
+        mr={5}
```

Comment:
Here we have `mr=5` and in the other case `mr=2` (so they are inconsistent). Can we remove them and specify the spacing as `gap={2}` on the element from line 77?